interfaces/automated-testing/geo-awareness/geo-awareness.yaml

```diff
@@ -147,57 +147,100 @@ components:
       required:
         - uomDimensions
         - verticalReferenceType
-        - verticalReference
-        - longitude
-        - latitude
       properties:
         uomDimensions:
           $ref: '#/components/schemas/UomDimensions'
         verticalReferenceType:
           $ref: '#/components/schemas/VerticalReferenceType'
         verticalReference:
+          description: >-
+            Height above vertical reference datum, in units of uomDimensions.
           type: number
+          default: 0
```

Comment:
Some serialization protocols can't differentiate between omission and the default value of a field (Protobuf).  IMO, this was a very questionable design decision, but they are in use so it's generally best to accommodate their limitations when practical.  Best practice for this is generally to make otherwise-required fields optional with a default value that matches the standard default value for that data type.